adders/routify/config/tests.js

```diff
@@ -12,10 +12,10 @@ export const tests = defineAdderTests({
                 await elementExists(".routify-demo");
                 await expectUrlPath("/");
 
-                await click(".routify-demo .demo", true);
+                await click(".routify-demo .demo", "/demo");
                 await expectUrlPath("/demo");
 
-                await click(".routify-demo .index", true);
+                await click(".routify-demo .index", "/");
                 await expectUrlPath("/");
```

Comment:
I wasn't 100% sure about this, but perhaps we can combine these cases? e.g. if a `path` is passed to `click`, then `click` will run `expectUrlPath` for you since `click` already knows the expected path

Reply:
Yes, I think we could, especially now that we need to pass the path a second time. But I think the current way is more explicit, and does not produce unwanted side effects.